extensions/escape-game/standalone.mjs

```diff
@@ -124,6 +146,33 @@ class Standalone {
     };
   }
 
+  async e2s_openDoor() {
+    var res = await this.helper.varUtils.writeVar("@doorLastOpen", null, null, Date.now().toString());
+    await this.helper.broadcastCellSetUpdateToAllUser(
+      'update',
+      "world1",
+      CellSet.fromObject({
+        name: LAYER_DOOR.layerName,
+        cells: []
+      }),
+    );
+    setTimeout(async function(obj) {
+      var lastOpen = await obj.helper.varUtils.readVar("@doorLastOpen", null, null);
+      lastOpen = parseInt(lastOpen);
+      if (lastOpen + 3000 <= Date.now()) {        
```

Comment:
Make this a config? Try this:
```
const config = require('config');
//...
const openTime = config.get('escape-game.doorOpenTime');
if (lastOpen + openTime <= Date.now()) { // ...
```

Then, please remember to adjust /run/config/default.json and /run/config/production.json.
If possible, adjust the corresponding config in hitcon-cat-adventures as well.